odc_index/requirements.txt

```
@@ -11,4 +11,4 @@ odc-thredds
 odc-apps-cloud
 # Pinned for Alex to index STAC
 sat-search==0.3.0-rc1
```

This can be set to `0.3.0` too, without the RC.

```suggestion
sat-search==0.3.0
```